client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -435,11 +425,12 @@ class Upload extends Component {
 				></NavigationHeader>
 
 				<HeaderCake backHref={ backPath }>{ translate( 'Install theme' ) }</HeaderCake>
-				{ ! showTrialAcknowledgeModal && ! isAtomic && (
+				{ ! showTrialAcknowledgeModal && this.props.isTransferInProgress && (
 					<HostingActivateStatus
 						context="theme"
 						onTick={ this.requestUpdatedSiteData }
 						keepAlive={ hasRequestedTrial && ! isAtomic }
+						forceEnable={ this.props.isTransferInProgress }
```

Comment:
`keepAlive` and `forceEnable` seem to have the same purpose. Can you join them?
```suggestion
						keepAlive={ ( hasRequestedTrial && ! isAtomic ) || this.props.isTransferInProgress }
```

Reply:
yeah, they are similar, but `keepAlive` is guarded by the `isTransferCompleted` constant from `HostingActivateStatus`. With this new prop we are forcing the state instead of using the one from `HostingActivateStatus`. 

I've contemplated using directly the `Notice` component, but I thought it makes sense to use the same  `HostingActivateStatus` component everywhere (it should be easier to refactor changes in the future this way) so that we can get a list of all places where AT notices are displayed.